install.sh

```diff
@@ -1122,7 +1269,7 @@ sandcats_register_name() {
   SANDCATS_CURL_PARAMS="${OVERRIDE_SANDCATS_CURL_PARAMS:-}"
 
   echo "Choose your desired Sandcats subdomain (alphanumeric, max 20 characters)."
-  echo "Type the word none to skip this step."
+  echo "Type the word none to skip this step, or help for help."
```

Comment:
FWIW I played with the text here for a while, unsure exactly how to phrase this. I am reasonably happy with this, having tried a number of options.